rdrf/rdrf/checks/security.py

```diff
@@ -0,0 +1,25 @@
+from django.core.checks import Error, register
+from django.urls import get_resolver, URLPattern, URLResolver
+from django.conf import settings
+
+
+@register()
```

Comment:
I thought we agreed that you will use `deploy=True` here.

That will allow developers developing locally without killing their `runserver`, but it will ensure they didn't forget about securing those views when the CI/CD kicks in.

Can you please also ensure, as part of this task or a separate follow-up task, that `django-admin check --deploy` is called from `buildspec.yml` CI/CD?